### ISSUE TITLE
[Feature Request]: Show menu icon only when activity triggered

### ISSUE THREAD

**Issue by eplusg** [jordanbaird/Ice#425]
### Check Existing Issues

- [X] I have checked existing issues, and this issue is not a duplicate

### Description

The old Bartender app included a trigger/activity feature that allowed users to hide certain apps until they became active.
They removed this for unknown reasons and I no longer use that app anyway since Ice works just fine!

I would like to suggest to add this feature to Ice if possible, here are two examples when it could be used:

Example 1 : Microsoft OneDrive (or any other sync service) may not be essential to display while it's idle, but when synchronization begins, it becomes relevant to see the icon in the menu bar to see that the application sync started. Once the sync is complete then the app can be hidden again.

Example 2: When the Mac is plugged in, the battery icon is not very important and takes up space. However, when the Mac is running on battery power, the icon becomes crucial for monitoring the remaining battery life and should be visible (or with an option to show it when. the battery reaches a certain percentage).

I guess there are many more use cases than these two but I think you get the point :)

It would be fantastic if such a feature could be implemented!

**Comment by eplusg** [comment]
I just realized, upon checking again, that there was already a similar request made.

https://github.com/jordanbaird/Ice/issues/317

**Comment by jordanbaird** [comment]
Duplicate of #62
